src/components/toolbar/__tests__/__snapshots__/toolbarFieldExport.test.js.snap

```diff
@@ -0,0 +1,132 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ToolbarFieldExport Component should aggregate export service calls: createExport 1`] = `
+[
+  [
+    {
+      "lorem": "ipsum",
+    },
+    {
+      "poll": {
+        "validate": [Function],
+      },
+    },
+  ],
+]
+`;
+
+exports[`ToolbarFieldExport Component should aggregate export service calls: getExport 1`] = `
+[
+  [
+    "dolorSit",
+  ],
+]
+`;
+
+exports[`ToolbarFieldExport Component should aggregate export service calls: getStatus 1`] = `
+[
+  [
+    {
+      "poll": {
+        "validate": [Function],
+      },
+    },
+  ],
+]
+`;
+
+exports[`ToolbarFieldExport Component should aggregate export service calls: getStatus, existing polling 1`] = `[]`;
+
+exports[`ToolbarFieldExport Component should aggregate export status, polling status with a hook: status, basic 1`] = `
+{
+  "isCompleted": undefined,
+  "isPolling": undefined,
+  "isProductPolling": false,
+  "productPollingFormats": [],
+}
+`;
+
+exports[`ToolbarFieldExport Component should aggregate export status, polling status with a hook: status, completed 1`] = `
+{
+  "isCompleted": undefined,
+  "isPolling": undefined,
+  "isProductPolling": false,
+  "productPollingFormats": [],
+}
+`;
+
+exports[`ToolbarFieldExport Component should aggregate export status, polling status with a hook: status, polling 1`] = `
+{
+  "isCompleted": undefined,
+  "isPolling": true,
+  "isProductPolling": true,
+  "productPollingFormats": [
```

Comment:
required:
this should be a flat list of format values... `['dolorSit']` vs `[{ format: 'dolorSit' }]`

confirm and correct the logic for this